admin/base/settings/defaults.py

```diff
@@ -23,9 +25,30 @@
     '.osf.io'
 ]
 
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+        'OPTIONS': {
+            'min_length': 5,
```

Comment:
@whit537 I should have added a comment.
This only works in Django 1.9, develop currently has 1.8